runelite-api/src/main/java/net/runelite/api/Constants.java

```diff
@@ -67,4 +67,11 @@
 	 * the plane value to 0-3.
 	 */
 	public static final int MAX_Z = 4;
+	/**
```

Comment:
Add new line above

Reply:
The rest of the Constants.java file was also formatted without new lines between each java doc comment so I added them into the whole file. I presume that's preferred?